src/text/font.rs

```diff
@@ -42,11 +43,22 @@ impl Font {
     }
 
     /// Creates and returns a monospace system font at the specified size and weight
+    ///
+    /// # Support
+    ///
+    /// The `monospace` font feature is available from version `10.15`.
+    ///
+    /// If the current system is using an older version the `monospacedSystemFontOfSize`
+    /// option will be omitted.
     pub fn monospace(size: f64, weight: f64) -> Self {
-        let size = size as CGFloat;
-        let weight = weight as CGFloat;
+        if os::is_minimum_semversion(10, 15, 0) {
+            let size = size as CGFloat;
+            let weight = weight as CGFloat;
 
-        Font(unsafe { ShareId::from_ptr(msg_send![class!(NSFont), monospacedSystemFontOfSize: size weight: weight]) })
+            Font(unsafe { ShareId::from_ptr(msg_send![class!(NSFont), monospacedSystemFontOfSize: size weight: weight]) })
+        } else {
+            Font(unsafe { ShareId::from_ptr(msg_send![class!(NSFont), weight: weight]) })
```

Comment:
I think you wanted `systemFontOfSize: size weight: weight`  here, no?

Reply:
Actually yes, this: https://developer.apple.com/documentation/appkit/nsfont/1524930-systemfontofsize?language=objc